portfolio/src/main/webapp/style.css

```diff
@@ -381,6 +399,11 @@ footer p{
 }
 
 
+/*PokeHood CSS*/
+
+/*Robotics CSS*/
+
```

Comment:
In general, I recommend to always check in html and css together. Prefer "half html + half css per pr" to "html in pr#1 and css in pr#2"

The idea is to avoid "broken" state on the master branch. You may submit html first, go through a weekend/long holiday, and come back to the project later only find out you forget which css needs to be implemented.

Reply:
Ok that sounds like a plan. I don't truly know how much work is good for a pull request so I just thought I complete on section and push then do the CSS and push. I don't want to make "empty" pull request at least in my mind but maybe this advice will help me get more out still doing the same amount of work.